### DISCUSSION TITLE
Repository Not Found | ggml-model-q4_0.bin

### DISCUSSION THREAD

**Discussion by 4ck4t4ny** [imartinez/privateGPT#330] [Q&A] [answered]
Hi there

Seems like there is no download access to "ggml-model-q4_0.bin".
I also logged in to huggingface and checked again - no joy.

Here are my .env settings:

PERSIST_DIRECTORY=db
MODEL_TYPE=GPT4All
MODEL_PATH=models/ggml-gpt4all-j-v1.3-groovy.bin
EMBEDDINGS_MODEL_NAME=ggml-model-q4_0.bin
MODEL_N_CTX=1000
 

### Detailed Error Message:
"Repository Not Found for url: https://huggingface.co/api/models/sentence-transformers/ggml-model-q4_0.bin.
Please make sure you specified the correct `repo_id` and `repo_type`.
If you are trying to access a private or gated repo, make sure you are authenticated.
Invalid username or password"

Any ideas welcome.

Thanks a bunch!

**Comment by imartinez** [comment, marked as answer]
We moved to a different and better embeddings model. Pull latest changes and review the example.env. You may want to delete your current .env and re-create it based on example.env which is already pointing to the right embeddings model. 

Sorry for the breaking changes... Everything is moving so fast that it is just impossible to stabilize just yet, would slow down the progress too much.

**Reply by 4ck4t4ny** [reply]
Hi
No prob, I fully understand.
Thanx for taking the time to answer this 👍
